### PR TITLE
[Merged by Bors] - Not me ... us

### PR DESCRIPTION
**Pull request by cart**
I don't see much of a reason at this point to boost my name over anyone elses. We are all Bevy Contributors.

### CODE REVIEW

**Comment by DJMcNab** [comment]
I think that the new recommendation is just not to have an authors field at all, for various reasons (primarily deadnaming concerns, I think)

Afaik it's no longer added by default by `cargo new`, for example.

**Comment by bjorn3** [comment]
https://rust-lang.github.io/rfcs/3052-optional-authors-field.html

**Comment by cart** [comment]
Ooh I dig it. Less boilerplate is good.

**Review by alice-i-cecile — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -258,9 +258,8 @@ To locally lint your files using the same workflow as our CI:
 
 If you end up adding a new official Bevy crate to the `bevy` repo:
```
Comment (DJMcNab) [inline CONTRIBUTING.md:260]: We still need to make sure the other metadata is correct -  at `github.com/bevyengine/bevy` as the url, etc.

(For example, `benches` doesn't have that which is probably *fine*)

**Review by DJMcNab — APPROVED** [review]
I can confirm that the only lines removed were the `authors` field.

File: CONTRIBUTING.md (hunk 1 above)
Comment (cart) [reply CONTRIBUTING.md:260]: Thats fair, but also a bit tangential. Feel free to create a new PR / issue :smile:

**Pull request closed without merging**